lib/plugins/create/templates/kubeless-python/.gitignore

```diff
@@ -0,0 +1,6 @@
+# package directories
```

Comment:
This file needs to be named `gitignore` (without a `.`). See --> https://github.com/serverless/serverless/tree/master/lib/plugins/create/templates/aws-nodejs

We had an issue with `.gitignore` and `.npmignore` and decided to add an extra step which updates the file from `gitignore` to `.gitgnore`.